vector/src/main/scala/geotrellis/vector/Results.scala

```diff
@@ -780,6 +782,8 @@ case class PointResult(geom: Point) extends GeometryResult
     with MultiLineStringMultiLineStringDifferenceResult
     with MultiPointMultiPointDifferenceResult
     with MultiPolygonMultiPolygonDifferenceResult
+    with MultiPointMultiPolygonUnionResult
+    with MultiPointMultiLineStringUnionResult
```

Comment:
Paging @jpolchlo -- seems like JTS got a bit better at simplifying the results of union. I was pretty mechanical about this, could use your :eyes: as the vector master.

Reply:
Happy to look.  Got any specific concerns?  These Result types are pretty labyrinthine.